src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -128,6 +131,7 @@ def load_arguments(self, _):
         c.argument('account_name', account_name_type, id_part=None)
         c.argument('database_name', options_list=['--name', '-n'], help="Database name")
         c.argument('throughput', help='The throughput of SQL database (RU/s). Default value is 400')
+        c.argument('max_throughput', help='The maximum throughput of SQL database can scale to (RU/s). Provided when the container is autoscale enabled.')
```

Comment:
Is there a rang for max throughput?

Reply:
no there is not, it should start with4k, that's all.